alibi_detect/cd/pytorch/lsdd_online.py

```diff
@@ -114,6 +115,7 @@ def _configure_normalization(self, eps: float = 1e-12):
         x_ref_means = x_ref.mean(0)
         x_ref_stds = x_ref.std(0)
         self._normalize = lambda x: (x - x_ref_means) / (x_ref_stds + eps)
+        self._unnormalize = lambda x: (torch.as_tensor(x) * (x_ref_stds + eps) + x_ref_means).cpu().numpy()
```

Comment:
I didn't notice this. I've done some digging and think the difference seems to be due to where they are used:

- `LSDDDriftOnline`: `_normalize` only used within the torch backend `score` method:
https://github.com/SeldonIO/alibi-detect/blob/1af9fc053ac224a599f16fd78e0d894de12f89f1/alibi_detect/cd/pytorch/lsdd_online.py#L205-L207
   Therefore, `_normalize` always recieves a `torch.Tensor`.
   
- `LSDDDrift`: `_normalize` is used in the `torch` backend `score` method, but also in the base `predict` method (in order to serve the `update_reference` functionality:
https://github.com/SeldonIO/alibi-detect/blob/1af9fc053ac224a599f16fd78e0d894de12f89f1/alibi_detect/cd/base.py#L797-L803
   In this case, `_normalize` recieves a `np.ndarray`. The type signature of `_configure_normalize` is still correct however, since this is called within the torch backend hence recieves a `torch.Tensor`.

Reply:
For the `_unnormalize` functions it's a similar story. The `x_ref_means` and `x_ref_stds` are `torch.Tensor`'s, but `_unnormalize` is called in the top-level `LSDDDrift.get_config`, hence `x` is a `np.ndarray` and needs converting back to `torch.Tensor`.